packages/tao/src/commands/run.ts

```diff
@@ -263,19 +263,32 @@ async function runExecutorInternal<T extends { success: boolean }>(
       );
     }
   } else {
+    let errors = [];
+
     require('../compat/compat');
-    const observable = await (
-      await import('./ngcli-adapter')
-    ).scheduleTarget(
-      root,
-      {
-        project,
-        target,
-        configuration,
-        runOptions: combinedOptions,
-        executor: targetConfig.executor,
-      },
-      isVerbose
+    const observable = (
+      await (
+        await import('./ngcli-adapter')
+      ).scheduleTarget(
+        root,
+        {
+          project,
+          target,
+          configuration,
+          runOptions: combinedOptions,
+          executor: targetConfig.executor,
+        },
+        isVerbose
+      )
+    ).pipe(
+      (await import('rxjs/operators')).tap(
+        ({ success, error }) => (!success && error ? errors.push(error) : 0),
+        (error) => (error.message ? errors.push(error.message) : 0),
```

Comment:
I don't think we need an error handler at all here. Those will be handled properly already.